flexget/plugins/sites/descargas2020.py

```diff
@@ -67,13 +67,10 @@ def parse_download_page(self, url, task):
                 torrent_id = match.group(1)
         if not torrent_id:
             log.debug('torrent ID not found, searching openTorrent script')
-            torrent_id_prog = re.compile('function openTorrent.*\n.*\{.*(\n.*)+window\.location\.href =\s*\"(.*\/\d+_-.*[^\/])\/?\";')
-            torrent_ids = soup.findAll(text=torrent_id_prog)
-            if torrent_ids:
-                match = torrent_id_prog.search(torrent_ids[0])
-                if match:
-                    torrent_id = match.group(2)
-                    return torrent_id.replace('descargar-torrent', 'download') + '.torrent'
+            match = re.search('function openTorrent.*\n.*\{.*(\n.*)+window\.location\.href =\s*\".*\/(\d+.*)\";',
+                              page.text, re.MULTILINE)
+            if match and len(match.groups())>1:
```

Comment:
If there is a match, it should always contain two capture groups, so this check seems unnecessary...

Reply:
Indeed! Fixed